advancedlog/logger.go

```diff
@@ -0,0 +1,60 @@
+package logger
+
+import "os"
+
+const (
+	// OutputFormatKey ...
+	OutputFormatKey = "output-format"
```

Comment:
Do we need this here?

Reply:
No, these are the parameters for the cli. I forgot to remove them when I moved the logger into this package.